bigquery/query_join_your_data.md

```diff
@@ -8,9 +8,9 @@ grand_parent: BigQuery
 
 # Query Category: Joining with Your Own Data
 
-* Using FIPS code:
+### Using [FIPS](https://www.census.gov/library/reference/code-lists/ansi.html) codes
 
-Join with Fatal Accidents report from National Highway Traffic Safety Administration to compute counties with highest fatal accidents per capita. Winner is Loving County, TX (the least populated county in main US), followed by Kenedy County, TX.
+Join with Fatal Accidents report from National Highway Traffic Safety Administration (source: [Google Cloud Public Datasets](https://pantheon.corp.google.com/marketplace/product/nhtsa-data/nhtsa-traffic-fatalities?project=) to compute counties with highest fatal accidents per capita. Winner is Loving County, TX (the least populated county in main US), followed by Kenedy County, TX.
```

Comment:
This is also available on AH.  I think we should  s/pantheon.corp.google.com/console.cloud.google.com/

https://console.cloud.google.com/bigquery/analytics-hub/exchanges(analyticshub:projects/1057666841514/locations/us/dataExchanges/google_cloud_public_datasets_17e74966199/listings/nhtsa_traffic_fatalities_17f892354f9)

Reply:
Done